.storybook/preview.js

```diff
@@ -25,7 +25,7 @@ window.__setCurrentStory = function (categorization, story) {
 /** @type {NonNullable<import('@storybook/react').Story['decorators']>} */
 export const decorators = [withFluentProvider, withStrictMode];
 
-/** @type {import('@storybook/react').Parameters} */
+/** @type {import('@storybook/addons').Parameters} */
```

Comment:
This was something I noticed while working on the react-examples storybook types: `Parameters` from `@storybook/react` is a super generic string/any dictionary, whereas the one from `@storybook/addons` includes all our custom types.